### PR TITLE
client profile important notes (DEV-709)

### PR DESCRIPTION
**Pull request by Davit-BetterAngels**
DEV-709

### CODE REVIEW

**Review by mikefeldberg — APPROVED** [review]

**Review by shiozakiyuko — APPROVED** [review]
Looks good to me!

File: libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  scrollRef: RefObject<ScrollView>;
+}
+
+export default function ImportantNotes(props: IImportantNotesProps) {
+  const { scrollRef, expanded, setExpanded } = props;
+  const { control } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
+
+  const isImportantNotes = expanded === 'Important Notes';
```
Comment (tglaz) [inline libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx:23]: Seems like this `const` actually tracks whether Important Notes is currently expanded. Perhaps rename it to `isExpanded`?

File: libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  return (
+    <Accordion
+      scrollRef={scrollRef}
+      expanded={expanded}
+      setExpanded={() => {
+        setExpanded(isImportantNotes ? null : 'Important Notes');
```
Comment (tglaz) [inline libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx:29]: The `Important Notes` is being repeated (we check for it in the `isImportantNotes` const). 
Can we pull it out into a const, like `accordionKey` or something? The name is less important, but a typo can easily break this.

File: libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      }}
+      mb="xs"
+      title="Important Notes"
+    >
+      {isImportantNotes && (
+        <View
+          style={{
+            height: isImportantNotes ? 'auto' : 0,
```
Comment (tglaz) [inline libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx:37]: is this ternary for height necessary? 
Based on the check above looks like this wouldn't render at all if not `isImportantNotes`? (Unless this acts differently in react native)

**Review by tglaz — CHANGES_REQUESTED** [review]

File: libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx (hunk 3 above)
Comment (Davit-BetterAngels) [reply libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx:37]: react native has some weird stuff. its safe this way

File: libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx (hunk 2 above)
Comment (Davit-BetterAngels) [reply libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx:29]: good one. We have a code cleaning task can walk through this as well

File: libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx (hunk 1 above)
Comment (Davit-BetterAngels) [reply libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx:23]: everywhere is "expanded" we can do this in code cleaning as well. Not to touch all files for this PR

Comment (tglaz) [reply libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx:23]: created follow up ticket https://betterangels.atlassian.net/browse/DEV-861

**Pull request merged**